### DISCUSSION TITLE
Error running QERaman on MacOS: "executable file is not an executable"

### DISCUSSION THREAD

**Discussion by EnriqueHernandezHub** [nguyen-group/QERaman#12] [Q&A]
I am running the [graphene example](https://github.com/nguyen-group/QERaman/blob/main/examples/graphene/run.sh) on MacOS Sonoma. The compilation both for QE and QERaman worked fine (I needed to redefine `CPP` in `make.inc` as `CPP = gcc -E` for QE to compile in Mac, according to the [QE wiki](https://gitlab.com/QEF/q-e/-/wikis/Support/macos-error)). 

However, when I am running the [graphene example](https://github.com/nguyen-group/QERaman/blob/main/examples/graphene/run.sh), I get an error as follows when executing `$ mpirun ph_mat.x <ph.in> ph.out`: 

```
Program received signal SIGSEGV: Segmentation fault - invalid memory reference.
Backtrace for this error:
Could not print backtrace: executable file is not an executable
```

There is no further information in `ph.out`. I was able to run it in a different system (Linux), following the exact same steps both for the installation, compilation and the execution steps for the example. What am I missing out in my QERaman (or QE) compilation process for Mac?

**Comment by nguyen-group** [comment]
Hi,

Could you check with ph.x <ph.in> ph.out for an other example first?,
You can find some examples here: https://github.com/nguyen-group/QE-SSP

If it works, the QE is fine, then the problem might be in QERaman.

Best,
Nguyen
